app/assets/stylesheets/forever_style_guide/modules/_message_box.scss

```diff
@@ -0,0 +1,12 @@
+@mixin message_box($base-color) {
+  background-color: transparentize($base-color, 0.95);
+  border: 1px solid transparentize($base-color, 0.6);
+  border-radius: $border-radius-xl;
+}
+
+// creates a message_box class for each color in core color dictionary
+@each $id, $color in $core_colors {
```

Comment:
why limit this to core_colors? might be beneficial to be more flexible and use $color in $colors